WordPress/src/main/res/layout/prepublishing_tags_fragment.xml

```diff
@@ -4,20 +4,21 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
+    <include
+        android:id="@+id/prepublishing_toolbar"
+        layout="@layout/prepublishing_toolbar" />
+
     <com.google.android.material.textview.MaterialTextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/margin_large"
-        android:layout_marginTop="@dimen/margin_medium_large"
+        android:layout_gravity="center_horizontal"
```

Comment:
I don't hold a strong opinion, but I had the thought that this feels more natural if left-aligned. Feel free to disregard.

Reply:
I was torn about this, but opted to center it since the toolbar title is centered. If I left-align it, it looks like this:

![start](https://github.com/user-attachments/assets/5367b891-201b-4be2-917b-4ce4291d7396)

That looks a bit odd to me, but I'm not at all opposed to that if you think it looks fine.

Reply:
I think left aligned is better. If you can improve the vertical rhythm by adding more space above the description, that would help as well. 

![balance-vertical-rhythm](https://github.com/user-attachments/assets/7f57126a-7f29-447b-938e-98678f040239)

Reply:
Done!

![tags](https://github.com/user-attachments/assets/ceab83de-4f84-4375-9833-ac9c5c02d4b6)

Reply:
Thanks! 🚀